.eslintrc.yaml

```diff
@@ -23,7 +23,7 @@ rules:
   prefer-const: ["error", {"destructuring": "all"}]
 
   # Code style rules
-  # arrow-parens: ["error", "always"]
+  arrow-parens: ["error", "always"]
```

Comment:
Hmm. I'm not a fan of this rule and think it adds noise to functional composition, which is widely used in this codebase. (I'm probably responsible for most of the places this commit adds parens.)

I think `=>` is already an unambiguous marker of a function that the parens aren't needed.

Reply:
Would you be in favor of enforcing consistency with other options (i.e. b2c466bdda38f5fdf89286207112b4e1b323b81e instead of 82e6755e5a18c475fdcd3fd0697905f389af63b3), or just dropping this rule entirely?

Reply:
I'd tend towards dropping the rule, as it's stylistic, and I don't think consistency for consistency's sake adds much here.

Reply:
Replaced 82e6755e5a18c475fdcd3fd0697905f389af63b3 with bb530526987dbe774c220842f41812f970a2aacd.